Public/Src/Engine/Dll/InputTracker.cs

```diff
@@ -570,6 +572,23 @@ private void RegisterFile(string path, ContentHash hash)
                 }
             }
 
+            // Write the mounts that impact the build
+            if (mountsImpactingBuild == null)
+            {
+                writer.Write(0);

```

Comment:
tiny nitpick: Upon deserialization we can't distinguish empty mounts from null mounts...

Reply:
I don't think that's that big of a deal. But are you allowed to have a mount that has no name in the first place?

Reply:
I take that back, empty mount names don't make a difference here actually.

I'm just going to leave it as-is though to match the pattern that the environment variables dictionary above uses for serialization.

Reply:
sounds good, was merely observigin the difference

---
In reply to: [274223325](https://github.com/Microsoft/BuildXL/pull/126#discussion_r274223325) [](ancestors = 274223325)